package.json

```diff
@@ -16,14 +16,26 @@
   "scripts": {
     "preanalyze-bundles": "webpack --config webpack.browser.config.js --profile --json > stats.json",
     "analyze-bundles": "webpack-bundle-analyzer stats.json .",
-    "prestart": "npm run targets:standalone",
-    "start": "webpack-dev-server --config webpack.standalone.config.js",
-    "targets": "run-p targets:*",
-    "targets:browser": "run-p targets:browser:*",
-    "targets:browser:css": "node-sass src/form.scss --output-style compressed | postcss --config postcss.config.json -u autoprefixer -u postcss-custom-properties --no-map -o targets/browser/happychat.css",
-    "targets:browser:js": "webpack --config webpack.browser.config.js",
-    "targets:standalone": "webpack --config webpack.standalone.config.js",
-    "targets:wordpress": "webpack --config webpack.wordpress.config.js",
+    "start": "run-p dev:standalone",
+    "dev:standalone": "run-p dev:standalone:*",
+    "dev:standalone:js": "NODE_ENV=development webpack-dev-server --config webpack.standalone.config.js",
+    "dev:standalone:css": "node-sass src/form.scss | postcss --config postcss.config.json -u autoprefixer -u postcss-custom-properties -o targets/standalone/happychat.css",
+    "dev:browser": "run-p dev:browser:*",
+    "dev:browser:js": "NODE_ENV=development webpack-dev-server --config webpack.browser.config.js",
+    "dev:browser:css": "node-sass src/form.scss | postcss --config postcss.config.json -u autoprefixer -u postcss-custom-properties -o targets/browser/happychat.css",
+    "dev:wordpress": "run-p dev:wordpress:*",
+    "dev:wordpress:js": "NODE_ENV=development webpack-dev-server --config webpack.wordpress.config.js",
+    "dev:wordpress:css": "node-sass src/form.scss | postcss --config postcss.config.json -u autoprefixer -u postcss-custom-properties -o targets/wordpress/assets/happychat.css",
```

Comment:
Ah, I see now why you did https://github.com/Automattic/happychat-client/pull/177/files#diff-a084b794bc0759e7a6b77810e01874f2R11

Reply:
Are we using this local CSS files anywhere?

Reply:
While working on calypso integration I use the dev commands and it was very helpful to have local css served. Based on NODE_ENV the local css will be served by all `dev:*` commands which tends to be extremely helpful while developing when css updates are needed.